eng/tools/generator/config/options.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-package model
+package config
```

Comment:
What are the differences between `option.go` and `options.go`? Can we merge them?

Reply:
merged options.go in option.go